test/skimNtuple_HHbtag.cpp

```diff
@@ -170,6 +170,69 @@ void setScaleFactor(std::map<std::string, std::vector<float>>& result_, const st
 }
 
 
+// Legacy triggers take precedence over Single-Tau and MET triggers
+std::unordered_map<std::string,bool> assignTriggerRegions(int ptype, std::string period,
+														  float pt1, float pt2, float eta1, float eta2,
+														  float eleEtaMax, float muEtaMax, float tau_thresh)
+{
+  bool Legacy_region    = false;
+  bool SingleTau_region = false;
+  bool MET_region       = false;
+  
+  if (ptype == 0) { //mutau
+	bool SingleLepton_validity = false;
+	bool CrossLepton_validity  = false;
+	bool SingleTau_validity    = false;
+	
+	if(period=="2018") {
+	  SingleLepton_validity = pt1 >= 33. and eta1 < muEtaMax;
```

Comment:
If I understand this function this is the threshold for the single muon trigger of 2018 which should be 24+1 (or +2, I forgot what we agreed) so 25 or 26.

Reply:
good catch! I think 25 (also judging from the values for the other periods, which look correct)

Reply:
Ok I will continue reading then :)

Reply:
I was copying the values from my Python validation framework and KLUB, and I copied etau cuts instead of mutau. Thank you for noticing.